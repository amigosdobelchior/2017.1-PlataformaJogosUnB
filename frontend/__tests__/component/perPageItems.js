import React from 'react';
import {Dropdown} from "semantic-ui-react";
import renderer from 'react-test-renderer';
var expect = required('expect');

test('Test render PerPageItems', () => {
  const component = renderer.create(
      <Dropdown text="name" selection>
          <Dropdown.Menu>
              ['Drop1', 'Drop2', 'Drop3']
          </Dropdown.Menu>
      </Dropdown>
	);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
