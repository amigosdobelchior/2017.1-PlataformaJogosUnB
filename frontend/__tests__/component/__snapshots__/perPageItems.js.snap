// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test render PerPageItems 1`] = `
<div
  aria-busy={undefined}
  aria-disabled={undefined}
  aria-expanded={false}
  aria-multiselectable={undefined}
  className="ui selection dropdown"
  onBlur={[Function]}
  onChange={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onMouseDown={[Function]}
  role="listbox"
  tabIndex={0}
>
  <select
    aria-hidden="true"
    multiple={undefined}
    name={undefined}
    type="hidden"
    value=""
  >
    <option
      value=""
    />
  </select>
  <div
    className="text"
  >
    name
  </div>
  <i
    aria-hidden="true"
    className="dropdown icon"
  />
  <div
    className="menu transition"
  >
    [
    Drop1
    , 
    Drop2
    , 
    Drop3
    ]
  </div>
</div>
`;
